### DISCUSSION TITLE
USB-C端子を接続していない方のLEDが正しく点灯しない

### DISCUSSION THREAD

**Discussion by FiNGAHOLiC** [Yowkees/keyball#399] [Q&A] [answered]
お世話になります。

掲題の件ですが、
左右同じエフェクトで点灯していたLEDがしばらく放置していると、
以下写真のようにUSB-C端子に接続していない側のLEDが同期しなくなる現象（USB端子を接続している側が意図するエフェクト）が起こります。また、その際に同記しなくなった側のキー入力が一切受け付けなくなります。
上記の状態を放置していると再度同期し始めます。
ちなみにUSB-C接続を左右逆にしても同様の症状です。

※LEDエフェクトはデフォルトでレイヤー3に設定されているRGB Mode Rainbowを使用しています
※USB-C端子に接続していない側のLEDは接続してる側と比較すると少しチカチカしている？

![IMG_7034](https://github.com/Yowkees/keyball/assets/246698/aa4d16f8-082e-452f-9697-085add716500)

過去のDiscussionを見ているとTRRSケーブルの不具合も考えられるのですが、
はんだ付けを見直しても問題点がよく分からなかったので質問させていただきました。

こちら原因や見落としている過去事例があればご教示いただきたく、
ご確認いただければ幸いです。

| 左 | 右 |
| - | - |
| ![IMG_7035](https://github.com/Yowkees/keyball/assets/246698/6f956869-da3e-4024-b650-e37ac357c406) | ![IMG_7036](https://github.com/Yowkees/keyball/assets/246698/aa14bd63-6809-428f-b953-d40229efe04f) |

**Comment by FiNGAHOLiC** [comment]
動画も添付します。
動画25秒から、トラックボールを触り始めた時から上記の症状が始まります。

https://github.com/Yowkees/keyball/assets/246698/73d6ad48-f70b-4a83-b227-e20cc8efd29e

**Comment by Yowkees** [comment, marked as answer]
ご連絡ありがとうございます。

おそらくTRSコネクタの接触不良が原因と考えられます。
例えば、Keyballを使用中にTRSコネクタを回転させても同様の症状となることがあります。

またLEDのちらつきですが、こちらは私の手持ちのものでも起こります。コンスルーの1ピンあたりの電流定格が1Aですので、LEDの数量的に結構限界に近く、白っぽい色で光量を最大とするとちらつきが見られますので、色味や光量を調整いただければ幸いです。

以上、お手数ですがご確認お願いいたします。

**Comment by FiNGAHOLiC** [comment]
@Yowkees ご回答ありがとうございます。

TRRSケーブルを変更してみたところ、同期しない問題は今のところ発生しなくなりました。
ご指摘ありがとうございます。

> 色味や光量を調整いただければ幸いです

なるほど、こちら承知しました。
色々設定を変更してみます。

**Comment by Yowkees** [comment]
不具合が解消したようでよかったです。安心しました。
ぜひ調整してご使用いただければ幸いです。
